src/components/Button/index.tsx

```diff
@@ -30,18 +30,18 @@ const buttonStyles = cva(
       {
         variant: "solid",
         colorscheme: "primary",
-        className: "bg-primary-500 hover:bg-primary-600",
+        className: "bg-primary hover:bg-primary",
       },
       {
         variant: "outline",
         colorscheme: "primary",
         className:
-          "text-primary-600 border-primary-500 bg-transparent hover:bg-primary-100",
+          "text-primary border-primary bg-transparent hover:bg-primary",
       },
       {
         variant: "ghost",
         colorscheme: "primary",
-        className: "text-primary-600 bg-transparent hover:bg-primary-100",
+        className: "text-primary bg-transparent hover:bg-primary",
```

Comment:
From the PR description, it seems like this PR intends to add `Avatar` component. Do you mind explaining the rationale behind changing the styles of the `Button` component here?

Reply:
The reason behind it that we have define our own theme in tailwind config file according to that we don't have "primary-600" in theme so if don't change storybook will not show css on avatar component. This button is not defined by me it come with storybook default boilerplate.